openweathermap-ios/.swiftlint.yml

```diff
@@ -0,0 +1,41 @@
+whitelist_rules:
+  - closure_spacing
+  - colon
+  - empty_enum_arguments
+  - fatal_error_message
+  - force_cast
+  - force_try
+  - force_unwrapping
+  - implicitly_unwrapped_optional
+  - legacy_cggeometry_functions
+  - legacy_constant
+  - legacy_constructor
+  - legacy_nsgeometry_functions
+  - operator_usage_whitespace
+  - redundant_string_enum_value
+  - redundant_void_return
+  - return_arrow_whitespace
+  - trailing_newline
+  - type_name
+  - unused_closure_parameter
+  - unused_optional_binding
+  - vertical_whitespace
+  - void_return
+  - custom_rules
+  - indentation
+
+excluded:
+  - Carthage
+  - Pods
+
+colon:
+  apply_to_dictionaries: false
+
+indentation: 2
```

Comment:
Consider changing it to 4 because it is the Xcode default value and our project is already using it this way.

Reply:
Okay, indentation changed to four.